wiki/Article_styling_criteria/Writing/en.md

```diff
@@ -83,7 +83,7 @@ When writing out game modifiers for tournament articles, they must instead use u
 Gameplay elements must not be capitalised, unless they act as a title for a link that points to the article. The following is an example:
 
 ```markdown
-In osu!standard, beatmaps are composed of three different gameplay elements: hit circles, sliders, and spinners.
+In osu!, beatmaps are composed of three different gameplay elements: hit circles, sliders, and spinners.
```

Comment:
```suggestion
In osu! mode, beatmaps are composed of three different gameplay elements: hit circles, sliders, and spinners.
```

Reply:
Now needs definite article because English is weird
"In the osu! mode"

Reply:
Yes, it does look better.

Reply:
decided to go with `the osu! game mode` as most pages (the Ranking Criteria, the World Cup articles, etc) are writing it this way instead

![image](https://user-images.githubusercontent.com/36564236/114734980-f2c6a480-9d6e-11eb-9d4e-cf50623a0c40.png)

Reply:
That's what I did here: #5255 

However, in some cases this is not applicable, so I left `osu! mode`.